src/dodal/beamlines/i13_1.py

```diff
@@ -19,7 +20,8 @@
 set_path_provider(
     StaticVisitPathProvider(
         BL,
-        Path("/data/2024/cm37257-4/"),  # latest commissioning visit
+        Path("/dls/i13-1/data/2024/cm37257-5/tmp/"),  # latest commissioning visit
+        client=LocalDirectoryServiceClient(),
```

Comment:
The local client is just incrementing from 1 and restarts with the bluesky service, you may want `RemoteDirectoryServiceClient("http://i13-1-control:8088/api")` which queries GDA for what the next scan number should be.

Reply:
Except I13-1's GDA is not yet configured with this functionality, so it will just break. I would suggest doing that as a separate PR after GDA is upgraded

Reply:
I thought that part of the REST API was always available?

Reply:
No, it has to be configured